addons/enemybar/gui_settings.lua

```diff
@@ -0,0 +1,251 @@
+stbg_body_settings.visible = true
+stbg_body_settings.color = {}
+stbg_body_settings.color.alpha = 255
+stbg_body_settings.color.red = 0
+stbg_body_settings.color.green = 51
+stbg_body_settings.color.blue = 255
+stbg_body_settings.size = {}
+stbg_body_settings.size.width = 0
+stbg_body_settings.size.height = 12
+stbg_body_settings.texture = {}
+stbg_body_settings.texture.path = bg_body_path
+stbg_body_settings.texture.fit = true
+stbg_body_settings.repeatable = {}
+stbg_body_settings.repeatable.x = 1
+stbg_body_settings.repeatable.y = 1
+stbg_body_settings.draggable = false
+
+tfgg_body_settings = {}
+tfgg_body_settings.pos = {}
+tfgg_body_settings.pos.x = settings.pos.x
+tfgg_body_settings.pos.y = settings.pos.y
+tfgg_body_settings.visible = true
+tfgg_body_settings.color = {}
+tfgg_body_settings.color.alpha = 200
+tfgg_body_settings.color.red = 255
+tfgg_body_settings.color.green = 0
+tfgg_body_settings.color.blue = 0
+tfgg_body_settings.size = {}
+tfgg_body_settings.size.width = 0
+tfgg_body_settings.size.height = 12
+tfgg_body_settings.texture = {}
+tfgg_body_settings.texture.path = fg_body_path
+tfgg_body_settings.texture.fit = true
+tfgg_body_settings.repeatable = {}
+tfgg_body_settings.repeatable.x = 1
+tfgg_body_settings.repeatable.y = 1
+tfgg_body_settings.draggable = false
+
+tfg_body_settings = {}
+tfg_body_settings.pos = {}
+tfg_body_settings.pos.x = settings.pos.x
+tfg_body_settings.pos.y = settings.pos.y
+tfg_body_settings.visible = true
+tfg_body_settings.color = {}
+tfg_body_settings.color.alpha = 255
+tfg_body_settings.color.red = 255
+tfg_body_settings.color.green = 51
+tfg_body_settings.color.blue = 0
+tfg_body_settings.size = {}
+tfg_body_settings.size.width = 0
+tfg_body_settings.size.height = 12
+tfg_body_settings.texture = {}
+tfg_body_settings.texture.path = fg_body_path
+tfg_body_settings.texture.fit = true
+tfg_body_settings.repeatable = {}
+tfg_body_settings.repeatable.x = 1
+tfg_body_settings.repeatable.y = 1
+tfg_body_settings.draggable = false
+
+stfg_body_settings = {}
+stfg_body_settings.pos = {}
+stfg_body_settings.pos.x = settings.pos.x
+stfg_body_settings.pos.y = settings.pos.y
+stfg_body_settings.visible = true
+stfg_body_settings.color = {}
+stfg_body_settings.color.alpha = 255
+stfg_body_settings.color.red = 0
+stfg_body_settings.color.green = 102
+stfg_body_settings.color.blue = 255
+stfg_body_settings.size = {}
+stfg_body_settings.size.width = 0
+stfg_body_settings.size.height = 12
+stfg_body_settings.texture = {}
+stfg_body_settings.texture.path = fg_body_path
+stfg_body_settings.texture.fit = true
+stfg_body_settings.repeatable = {}
+stfg_body_settings.repeatable.x = 1
+stfg_body_settings.repeatable.y = 1
+stfg_body_settings.draggable = false
+
+require('targetBar')
+require('subtargetBar')
+
+init_images = function()
+    init_target_images()
+    init_subtarget_images()		
+end
+
+check_claim = function(claim_id)
+    local player = windower.ffxi.get_player()
+    local party = windower.ffxi.get_party()
```

Comment:
You don't seem to need this variable in your code since you're using `windower.ffxi.get_mob_by_target` instead, so you might wanna remove it. `windower.ffxi.get_party` is a very expensive function call, so you should avoid it where possible. Currently you call it twice on each rendered frame.